### ISSUE TITLE
Feature request: UI: Improve visibility of selcted items in system tray menu

### ISSUE THREAD

**Issue by ags1234** [tnodir/fort#287]
The selected option in the system tray menu is hard to see as it is now, a trans-lucid, barely visible rectangle.
Exemplified in the attached picture.

Clear visibility is important in an UI. For the user it is not all the same if all traffic is blocked or all traffic is allowed.
The selected options should be clearly marked.

![2024-08-18_085951](https://github.com/user-attachments/assets/5007d6af-ec99-41b4-be7b-f9d18e9a7e9d)

**Comment by 03302024** [comment]
> The "Filter Mode" menu item has same icon as selected mode.

Yes, it took me a while to figure that out. On first glance, visiblity was poor.

**Comment by ags1234** [comment]
If visibility is poor because of Qt theme, maybe something can be done about the theme the software is shipped to the users with?
If not, maybe another theme?
Or maybe the menus can be enriched with an added check mark for the selected options?

It is baffling to me how efforts are being made to build this software with a very good control of all inbound and outbound traffic, only to end up, as a user, in the situation not to properly see how the firewall is configured.

**Comment by ags1234** [comment]
The missing icons in those sub-menus allows the user to see what is selected.
The fix is good. Thank you.
